[2024-01-27T12:44:15.678+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:40:16.216041+00:00 [queued]>
[2024-01-27T12:44:15.690+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:40:16.216041+00:00 [queued]>
[2024-01-27T12:44:15.691+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-27T12:44:15.713+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_database> on 2024-01-27 12:40:16.216041+00:00
[2024-01-27T12:44:15.730+0000] {standard_task_runner.py:60} INFO - Started process 34051 to run task
[2024-01-27T12:44:15.734+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sales_forecast_pipeline', 'load_to_database', 'manual__2024-01-27T12:40:16.216041+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/sales_forecast_pipeline.py', '--cfg-path', '/tmp/tmpagjm30ip']
[2024-01-27T12:44:15.735+0000] {standard_task_runner.py:88} INFO - Job 40: Subtask load_to_database
[2024-01-27T12:44:15.798+0000] {task_command.py:423} INFO - Running <TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:40:16.216041+00:00 [running]> on host fb667f005a5b
[2024-01-27T12:44:15.923+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_forecast_pipeline' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T12:40:16.216041+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-27T12:40:16.216041+00:00'
[2024-01-27T12:44:18.802+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-27T12:44:18.819+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sales_forecast_pipeline, task_id=load_to_database, execution_date=20240127T124016, start_date=20240127T124415, end_date=20240127T124418
[2024-01-27T12:44:18.915+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-27T12:44:18.938+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
